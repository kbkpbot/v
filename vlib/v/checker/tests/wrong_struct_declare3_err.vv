struct Foo {
}

struct Bar {
        __global foos []Foo
}
