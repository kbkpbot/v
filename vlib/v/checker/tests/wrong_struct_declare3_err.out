vlib/v/checker/tests/wrong_struct_declare3_err.vv:5:18: error: unexpected name `foos`, expecting `:`
    3 | 
    4 | struct Bar {
    5 |         __global foos []Foo
      |                  ~~~~
    6 | }
